pkg/ociinstaller/plugin.go

```diff
@@ -34,6 +34,11 @@ func InstallPlugin(ctx context.Context, imageRef string, sub chan struct{}) (*St
 
 	ref := NewSteampipeImageRef(imageRef)
 	imageDownloader := NewOciDownloader()
+	config := &pluginInstallConfig{}
+	for _, opt := range opts {
+		opt(config)
+	}
+	skipPluginConfig := config.skipConfigFile
```

Comment:
These lines should be the first lines of the func. (Not a biggie nor it is wrong, it's just how we have done it elsewhere 😄 )

Having another variable here seems excessive. You don't need another variable here `skipPluginConfig` (just use `config.skipConfigFile` wherever needed)